src/LinAlg/hiopVectorPar.cpp

```diff
@@ -795,6 +857,13 @@ bool hiopVectorPar::isfinite_local() const
   return true;
 }
 
+void hiopVectorPar::print()
```

Comment:
@cnpetra Add this function to easily display the vector in gdb. Maybe we should remove it from this PR.

Reply:
it does not hurt to have it in there